packages/webdriverio/src/commands/browser/custom$.ts

```diff
@@ -43,7 +44,7 @@ async function custom$ (strategyName, strategyArguments) {
     }
 
     if (res && typeof res[ELEMENT_KEY] === 'string') {
-        return await getElement.call(this, strategy, res)
+        return await getElement.call(this, strategy.toString(), res)
```

Comment:
shouldn't strategy be a string already?
(same for custom$$)

Reply:
nope, it is from type function as we receive it from a stored selector strategy (see line 29)

Reply:
anyway, casting it to string doesn't make much sense. `strategy` should be a string otherwise we should throw an error, imo @christian-bromann